subservers/manager.go

```diff
@@ -49,6 +54,9 @@ func (s *Manager) AddServer(ss SubServer, enable bool) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
+	// Register all sub-servers with the status server.
+	s.statusServer.RegisterSubServer(ss.Name())
```

Comment:
I guess this is another hint that we might want to merge the two managers instead.

Reply:
i think every subserver should have a status but not each thing that has a status is a subserver